test/password-field-validation.html

```diff
@@ -79,5 +84,23 @@
       expect(event.detail.valid).to.be.false;
     });
   });
+  describe('invalid cannot be set to false', () => {
+    let passwordField;
+
+    beforeEach(async() => {
+      passwordField = fixture('should-set-invalid');
```

Comment:
I'm wondering if `password-field` would be a better naming for the `basic` fixture after all. It would allow us to re-use it here then.

```js
describe('basic', () => {
  beforeEach(() => {
    passwordField = fixture('password-field');
  });
});

describe('invalid cannot be set to false', () => {
  beforeEach(() => {
    passwordField = fixture('password-field');
  });
});
```